examples/notebooks/beam-ml/run_inference_streaming_pipeline.ipynb

```diff
@@ -0,0 +1,292 @@
+      ],
+      "metadata": {
+        "id": "kH8SORNim8on"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "This notebook shows how to use the Apache Beam [RunInference](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.base.html#apache_beam.ml.inference.base.RunInference) transform in a streaming pipeline with [Google Cloud Pub-Sub](https://cloud.google.com/pubsub?utm_source=google&utm_medium=cpc&utm_campaign=na-US-all-en-dr-bkws-all-all-trial-b-dr-1605212&utm_content=text-ad-none-any-DEV_c-CRE_648329165516-ADGP_Desk%20%7C%20BKWS%20-%20BRO%20%7C%20Txt%20_%20Pub%2Fsub-KWID_43700075187144857-aud-664745643345%3Akwd-874320293016&utm_term=KW_pub%20sub%20google%20cloud-ST_pub%20sub%20google%20cloud&gclid=CjwKCAjw-IWkBhBTEiwA2exyO19xMFn6h1UKjb4QUavatV8Yb5Au9pCQj2_VAo0rzaYS8v2bq5VmuBoCL9wQAvD_BwE&gclsrc=aw.ds).\n",
+        "\n",
+        "This notebook demonstrates the following steps:\n",
+        "- Load and save a model from Hugging Face Models Hub.\n",
+        "- Use Pub/Sub IO in Python SDK as a streaming source.\n",
+        "- Use PyTorch model handler for RunInference.\n",
+        "\n",
+        "For more information about using RunInference, see [Get started with AI/ML pipelines](https://beam.apache.org/documentation/ml/overview/) in the Apache Beam documentation."
+      ],
+      "metadata": {
+        "id": "7N2XzwoA0k4L"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Before you begin\n",
+        "Set up your environment and download dependencies."
+      ],
+      "metadata": {
+        "id": "nhf_lOeEsO1C"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "wS9a3Y0oZ_l5"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install apache_beam[gcp]==2.48.0\n",
+        "!pip install torch\n",
+        "!pip install transformers\n",
+        "!pip install tensorflow"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Authenticate with Google Cloud\n",
+        "This notebook relies on Google Cloud Pub/Sub as an input to the pipeline as well for writing out the results. To use your Google Cloud account, authenticate this notebook."
+      ],
+      "metadata": {
+        "id": "A0v-oGSfsTHh"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "metadata": {
+        "id": "hNslPJGil2Zc"
+      },
+      "execution_count": 63,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Import dependencies and set up your bucket\n",
+        "Use the following code to import dependencies and to set up your Google Cloud Storage bucket.\n",
+        "\n",
+        "Replace `MESSAGE_TOPIC` and `RESPONSE_TOPIC` with the Pub/Sub topics in your project.\n",
```

Comment:
We should put this after the code instead of before, just because that's the way our style guide says to do it.